src/main/java/com/artipie/asto/Storage.java

```diff
@@ -199,5 +199,10 @@ protected Wrap(final Storage delegate) {
         ) {
             return this.delegate.exclusively(key, operation);
         }
+
+        @Override
+        public final CompletableFuture<? extends Meta> metadata(final Key key) {
```

Comment:
We can cover this with a test

Reply:
Interface `Storage.Wrap` is not extended by a class in this repo. All `Storage` sub-classes directly implement `Storage`.

Reply:
I think we need some kind of consistency test to be sure that `Wrap` implements all of the methods declared in `Storage` interface, e.g. `class verify extends Storage.Wrap {}` in some test will fail to compile if we update `Storage` interface but forget to change its `Wrap` implementation.

Reply:
I see now